src/index.tsx

```diff
@@ -65,6 +67,16 @@ if (process.env.REACT_APP_MAINNET_ENV === 'production') {
   initGoogleAnalytics()
 }
 
+export const sentryRequestId = uuid()
+if (process.env.REACT_APP_MAINNET_ENV === 'production') {
+  Sentry.init({
+    dsn: 'https://0b4e0f2a75a04734ad380be0a8882886@o302165.ingest.sentry.io/6156652'
```

Comment:
please put this to env files.

Also, the URL is not correct

Reply:
Updated.